engine/debug/debug_party.asm

```diff
@@ -1,13 +1,5 @@
-; This function is a debugging feature to give the player Tsunekazu Ishihara's
-; favorite Pokemon. This is indicated by the overpowered Exeggutor, which
-; Ishihara (president of Creatures Inc.) said was his favorite Pokemon in an ABC
-; interview on February 8, 2000.
-; "Exeggutor is my favorite. That's because I was always using this character
-; while I was debugging the program."
-; http://www.ign.com/articles/2000/02/09/abc-news-pokamon-chat-transcript
```

Comment:
Stuff like this I agree shouldn't exist in this detail in the source code... however, it would be a great addition to the pokered wiki "Trivia" section, and perhaps link to the routine here.

Reply:
Would this suffice then?
```suggestion
; Then producer Tsunekazu Ishihara used an Exeggutor for play testing.
; He also considered Exeggutor his favorite Pokémon at the time.
; https://web.archive.org/web/20000607152840/http://pocket.ign.com/news/14973.html
; https://web.archive.org/web/20150909170921/http://www.rigelatin.net/copycat/media/print/np134.php
; It is the only debug party member shared with Red, Green, and Japanese Blue.
```

Reply:
`s/Then producer/The game producer/`. Also why change the URL, and why add two? No need to hard-code archive.org, when/if it becomes necessary people can look it up there, or archive.is, or wherever.

Reply:
This just repeats the comment further down, and changes one definitive URL to two with hard-coded archive.org noise. The whole point was his direct quote.

Reply:
Let's just remove this comment since you add one further down.

Reply:
Sure.